### ISSUE TITLE
Root Nemo hangs for too long when network share is unavailable

### ISSUE THREAD

**Issue by ghost** [linuxmint/nemo#1950]
```
 * Nemo version (nemo --version): 3.8.6
 * Is issue with desktop or windowed nemo?: windowed
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...): Mint 19 Cinnamon
 * Graphics hardware *and* driver used: not relevant
 * 32 or 64 bit: 64
 ```

**Issue**

&nbsp;&nbsp;&nbsp;&nbsp;When some (any) network share is offline, trying to opening Nemo as root (but not as non-root) results in a hang of some 20 30 seconds, i.e. no (root) Nemo window appears for that length of time. This gets in the way of my doing work.

**Steps to reproduce**

&nbsp;&nbsp;&nbsp;&nbsp;I suspect that the following steps will reproduce the problem. I write 'suspect' because I have not tried to reproduce the issue on a clean installation of Mint.

1. Set up a network share.
2. Wait until or make it the case that the share is unavailable.
3. Use any one of various means to run Nemo as root.

**Expected behaviour**

&nbsp;&nbsp;&nbsp;&nbsp;A short hang. Better: Nemo opening with a busy pointer. Better yet: the following. Nemo showing a busy pointer when - if and only if - I try to access an inaccessible network folder, and not hanging at all upon open.

**Other information**

&nbsp;&nbsp;&nbsp;&nbsp;I am happy to supply my systemD unit files and `fstab`. I would _really_ like this problem, which seems to me long-standing (though I could find no exact duplicate bug report), to go away.

&nbsp;&nbsp;&nbsp;&nbsp;PcManFM does not have the same problem (at least on my system).

&nbsp;&nbsp;&nbsp;&nbsp;Previously I have had problems shutting Mint down when network shares were inaccessible (as documented [here](https://github.com/linuxmint/nemo/issues/1812) on the Nemo tracker). Those problems seemed to be caused by systemd. To solve those problems, I set (or, as the case may, be altered) various systemd timeouts, and set various systemd settings. At present, the computer no longer has any (serious) problems shutting down. Yet, the problem with Nemo persists.

**Comment by CottonEaster** [comment]
I am still getting a >1 minute hang when there is no network and I open Nemo as root. I do hope that this is fixed in Mint 19.1 (though I have not tested it on 19.1, and using VirtualBox so to do might be hard). Not hanging - in fairly normal situations - is pretty basic functionality, if I may say so.

**Comment by icarter09** [comment]
Closing this issue since the original submitter of this ticket is no longer present on Github.
